lib/framework/theme/theme_loader.dart

```diff
@@ -139,16 +141,14 @@ mixin ThemeLoader {
   TextTheme _buildTextTheme([YamlMap? textTheme]) {
     final fontFamily = Utils.optionalString(textTheme?['fontFamily']);
 
-    late TextStyle defaultStyle;
+    TextStyle defaultStyle = const TextStyle(
+        fontFamily: 'Inter', fontWeight: FontWeight.w400, color: Colors.black);
     try {
-      if (fontFamily == null) throw Exception();
-      defaultStyle = GoogleFonts.getFont(fontFamily, color: Colors.black);
-    } catch (e) {
-      defaultStyle = const TextStyle(
-          fontFamily: 'Inter',
-          fontWeight: FontWeight.w400,
-          color: Colors.black);
-    }
+      if (fontFamily != null) {
+        defaultStyle =
```

Comment:
This can return null if fontFamily=giberish

Reply:
GoogleFonts.getFont throws an exception when the fontFamily does not exist, see here - https://pub.dev/documentation/google_fonts/latest/google_fonts/GoogleFonts/getFont.html

So this is not an issue as the code is already catching exceptions and ignoring them.